VK/VK/Presentation/News/Cells/PostAndImageTableViewCell.swift

```diff
@@ -61,7 +61,7 @@ extension PostAndImageTableViewCell: UICollectionViewDataSource {
     func collectionView(_ collectionView: UICollectionView, numberOfItemsInSection section: Int) -> Int {
```

Comment:
```suggestion
    func collectionView(_ collectionView: UICollectionView, numberOfItemsInSection section: Int) -> Int {
```
Под марку Public Methods